build_runner/test/generate/build_integration_test.dart

```diff
@@ -357,6 +357,90 @@ main(List<String> args) async {
         await expectBuildOutput(expectCopyInOutput: false);
       });
     });
+
+    group('--define overrides build.yaml', () {
+      String buildContent = '''
```

Comment:
[nit] maybe `final buildContent` or `const buildContent` so that it's clear this doesn't change?

Reply:
done